src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -950,6 +955,40 @@ private void copyKeyAndTitle() {
         }
     }
 
+    private void copyKeyAndLink() {
+        List<BibEntry> selectedEntries = mainTable.getSelectedEntries();
+        if (!selectedEntries.isEmpty()) {
+            storeCurrentEdit();
+
+            StringBuilder sb = new StringBuilder();
+
+            long copied = 0;
+            for (BibEntry entry : selectedEntries.stream().filter(BibEntry::hasCiteKey).collect(Collectors.toList())) {
+                String key = entry.getCiteKeyOptional().get();
+                String url = entry.getField(FieldName.URL).isPresent() ? entry.getField(FieldName.URL).get() : "";
```

Comment:
Hm, which sin does it have to have an empty url? 
Wouldn't it be better if you also filter if the url is not empty?